grove/pyqaoa/maxcut_qaoa.py

```diff
@@ -30,7 +30,7 @@ def print_fun(x):
     print x
 
 
-def maxcut_qaoa(graph, steps=1, rand_seed=None):
+def maxcut_qaoa(graph, steps=1, rand_seed=None, connection=None):
```

Comment:
I guess we can't just default this to `CXN` because that'll be evaluated too early.